app/styles/layouts/auth.css

```diff
@@ -17,6 +17,10 @@
     margin-bottom: 1.5rem;
 }
 
+.gh-signin .form-group.error {
```

Comment:
Yep, that was the desired behaviour last time we looked at the sign-in validation.

Is that behaviour easy to replicate with the new validation? It's probably more desirable to keep it as-is for now and look at changing the behaviour as part bigger design review.